scala/lambdas/ingest-flow-control/src/test/scala/uk/gov/nationalarchives/ingestflowcontrol/Helpers.scala

```diff
@@ -109,7 +113,7 @@ object Helpers {
           existing.map(_.name)
         }
 
-    override def sendTaskSuccess(token: String): IO[Unit] = {
+    override def sendTaskSuccess[T: Encoder](token: String, potentialOutput: Option[T]): IO[Unit] = {
       errors.raise(_.sendTaskSuccess, "Error sending task success to step function") >>
```

Comment:
1. `potentialOutput` is unused here
2. Should we add the token to the Error message?

Reply:
As this is just a test implementation, we don't need it but we have to match the signature. similarly, including the token in here won't be of any use. Its just there to test that, if a real `sendTaskSuccess` method fails, we make use of that error. The message here can be anything.